pkg/operator/controller/canary/controller.go

```diff
@@ -244,6 +241,14 @@ func (r *reconciler) startCanaryRoutePolling(stop <-chan struct{}) error {
 			return
 		}
 
+		// Don't attempt to probe if route is not actually admitted.
+		if !checkRouteAdmitted(route) {
+			if err := r.setCanaryNotAdmittedStatusCondition(); err != nil {
+				log.Error(err, "error updating canary status condition")
+			}
+			return
+		}
```

Comment:
This raises the question, Should we also set the status to "Unknown" in the case that the route doesn't exist?  We can leave that as a follow-up if you prefer.

Reply:
Even if you go unmanaged, doesn't the ingress controller still ensure the canary route? Is there a situation in which a user can delete the canary route and it not come back (at least try to come back)? I suppose you can never be too sure...